test/ExternalPathsWatch.test.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
```

Comment:
Move this tests in https://github.com/webpack/webpack-dev-server/blob/master/test/ContentBase.test.js